jupyterhub/templates/hub/deployment.yaml

```diff
@@ -58,6 +58,7 @@ spec:
       serviceAccountName: hub
       {{- end }}
       securityContext:
```

Comment:
Same here - can we move securityContext to values.yaml and let users override it completely? That would be more flexible, and require fewer overrides in the future.

Reply:
Somewhat agree, the proposal in #1677 is to be able to specify additional fields in order to integrate [py-spy](https://github.com/benfred/py-spy#how-do-i-run-py-spy-in-kubernetes), in this case `capabilities`. But if we can generically override everything in here via `values.yaml` like @betatim pointed out [here](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues/1491#issuecomment-559855666) then that would probably work for us as well.

Reply:
Actually this might not work for our case in #1677 because this is the pod `securityContext` and according to https://github.com/benfred/py-spy#how-do-i-run-py-spy-in-kubernetes we need to set `capabilities` in the `container` spec below.